tree/20_cfe_basics/cfengine/bundles.cf

```diff
@@ -39,14 +39,29 @@
 # the online docs
 #
 
-# For CFEngine Core: 3.5.1 to 3.5.x
+# For CFEngine Core: 3.7.0 to 3.7.x
 # Bundles
 
 ###################################################
 # If you find CFEngine useful, please consider    #
 # purchasing a commercial version of the software.#
 ###################################################
 
+bundle common bundles_common
+# @ignore
+{
+  vars:
+      "inputs" slist => { "$(this.promise_dirname)/paths.cf",
+                          "$(this.promise_dirname)/files.cf",
+                          "$(this.promise_dirname)/commands.cf" };
+}
+
+body file control
+# @ignore
+{
+      inputs => { @(bundles_common.inputs) };
+}
```

Comment:
shouldn't we remove these ? do you know if they conflict with our loading method ?